OpenRA.Mods.Common/Util.cs

```diff
@@ -175,5 +176,15 @@ public static int RandomDelay(World world, int[] range)
 
 			return world.SharedRandom.Next(range[0], range[1]);
 		}
+
+		// TODO: Investigate caching this or moving it to ActorMapInfo
+		public static WDist MinimumRequiredScanRadius(Ruleset rules, WeaponInfo wi, bool onlyBlockers = false)
+		{
+			var allMinimumRequiredScanRadius = rules.Actors.SelectMany(a => a.Value.TraitInfos<HealthInfo>()).Max(h => h.Shape.OuterRadius);
+			var blockerMinimumRequiredScanRadius = rules.Actors.Where(a => a.Value.HasTraitInfo<BlocksProjectilesInfo>())
+				.SelectMany(a => a.Value.TraitInfos<HealthInfo>()).Max(h => h.Shape.OuterRadius);
+
+			return onlyBlockers ? blockerMinimumRequiredScanRadius : allMinimumRequiredScanRadius;
```

Comment:
Both `blockerMinimumRequiredScanRadius` and `allMinimumRequiredScanRadius` are set up every time this helper is called, but only one is ever used at a time.  They are completely independent, so why not make this two separate helpers?